pylib/setup.py

```diff
@@ -19,12 +19,25 @@
 import sys
 import warnings
 from distutils.core import setup, Extension
+from setuptools.command.build_ext import build_ext
 
 
-def get_extensions():
-    if "--no-compile" in sys.argv:
```

Comment:
this part isn't working, when arguments are being passed from `build` or from `pip install` command

need to refactor this, since had to disable cython of copyutil, (it wasn't working)

but seem after those change it does work.... 😕

Reply:
interesting.. if it works, it works..